github/issues.go

```diff
@@ -44,6 +44,7 @@ type Issue struct {
 	Repository       *Repository       `json:"repository,omitempty"`
 	Reactions        *Reactions        `json:"reactions,omitempty"`
 	Assignees        []*User           `json:"assignees,omitempty"`
+	ClosedBy         *User             `json:"closed_by,omitempty"`
```

Comment:
> (Note that named structs themselves don't get accessors [intentionally]... there is no `GetUser` method, for example, because you can just do something like `resp.User.GetName()` for example and it will just work... even if `resp.User == nil`... does that answer the question?)

I see. That answers my question, thanks.